Automattic/jetpack
# Admin Page: change dev version notice link to an external link

#### Changes proposed in this Pull Request:
* Change the link in DevVersionNotice to an external link by passing the `external={true}` prop to `NoticeAction`.
* Add tests for `DevVersionNotice`.

#### Jetpack product discussion
* n/a

#### Does this pull request change what data or activity we track or use?
* no

#### Testing instructions:

1. Install this branch and activate Jetpack.
2. Navigate to wp-admin -> Jetpack.
3. Since you're using a development branch, you should see the development version notice at the top of the page. Verify that the external link icon is displayed next to the `Submit Beta Feedback` link.
4. Click the "Submit Beta feedback' link in the notice. Verify that the link opens in a new tab or window.


#### Proposed changelog entry for your changes:
* Admin page: change the dev notice link to an external link.
